dw-list-item.js

```diff
@@ -60,19 +180,63 @@ export class DwListItem extends LitElement {
       /**
        * Set to true to show twoLine item
        */
-      twoLine: { type: Boolean, reflect: true },
+      twoLine: { type: Boolean, reflect: true},
 
        /**
        * Shows disabled style when true
        */
-      disabled: { type: Boolean, reflect: true }
+      disabled: { type: Boolean, reflect: true },
+
     };
   }
 
+  constructor(){
+    super();
+
+    this.twoLine = false;
+    this.dense = false;
+    this.disabled = false;
+  }
+
   render() {
     return html`
-      
-    `;
+      <div class="list-item layout horizontal center" tabindex="0">
+
+        ${this.disabled ? '' : html`<dw-ripple></dw-ripple>`}
+
+        <!-- Leading icon -->
+        ${this.leadingIcon ? this._leadingIconTemplate : ''}
+
+        <!-- Item text -->
+        <span class="item-text-container layout vertical flex ellipses">
```

Comment:
Here, `span` shouldn't be used. It's not a block level element. Same as for `title1` and `title2` container must not be `span`.